src/frontend/apps/impress/package.json

```diff
@@ -18,6 +18,7 @@
     "@blocknote/core": "0.21.0",
     "@blocknote/mantine": "0.21.0",
     "@blocknote/react": "0.21.0",
+    "@blocknote/xl-multi-column": "0.21.0",
```

Comment:
Since it took us a while to review this PR, maybe update this deps to the latest version now (0.25.1)